README.md

```diff
@@ -93,7 +103,7 @@ tensorboard --logdir=./
 To test the trained model, run the following command:
 
 ```
-python -m swarm_rl.enjoy --algo=APPO --env=quadrotor_multi --replay_buffer_sample_prob=0 --continuous_actions_sample=False --quads_use_numba=False --train_dir=PATH_TO_PROJECT/swarm_rl/train_dir --experiments_root=EXPERIMENT_ROOT --experiment=EXPERIMENT_NAME
+python -m swarm_rl.enjoy --algo=APPO --env=quadrotor_multi --replay_buffer_sample_prob=0 --quads_use_numba=False --train_dir=PATH_TO_PROJECT/train_dir/RUN_NAME --experiment=EXPERIMENT_NAME
```

Comment:
Do we need to add RUN_NAME in train_dir?

Reply:
Yes, for example to run it on my device, I had to do 
--train_dir=/home/resl/Testing/quad-swarm-rl/train_dir/quads_multi_mix_baseline_8a_local_v116/quad_mix_baseline-8_mixed_/ 
--experiment=00_quad_mix_baseline-8_mixed_q.c.rew_5.0
Is there a better way to phrase that?

Reply:
One way to check that is to run experiments by using a runner script and check "experiment" and "train_dir" in the cfg.json file.